javaclient/src/main/bin/start_consumer.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
This file should be under `javaclient/sbin` instead.

Reply:
i decided to remove the shell